commander/src/commands/transaction/sign.ts

```diff
@@ -157,6 +158,6 @@ export default class SignCommand extends BaseCommand {
 			throw errors;
 		}
 
-		this.print(removeUndefinedValues(txInstance.toJSON() as object));
+		this.print(removeUndefinedValues({} as object));
```

Comment:
Remove undefined values from which object?

Reply:
same as above, currently transaction cannot be signed by the JSON, and with migration, this needs to be re-written.
Also, transaction instance does not accept JSON value, so it cannot instantiate now 😢